src/components/Button/Button.tsx

```diff
@@ -263,9 +263,9 @@ export function Button({
         onClick={toggleDisclosureActive}
         onMouseUp={handleMouseUpByBlurring}
       >
-        <IconWrapper>
+        <span className={styles.Icon}>
```

Comment:
This is the same markup applied from `IconWrapper`

Reply:
Hmmm yeah, was getting this while trying to `yarn run build-consumer web`. [Fixed by moving the `IconWrapper` component into our usual subcomponent file structure (`/components/IconWrapper`)](https://github.com/Shopify/polaris-react/compare/fix-esnext-error?expand=1)

Reply:
It's the weirdest thing since there's nothing invalid about it :/ I was talking to Dan about this and he mentioned `IconWrapper` was added for testing purposes, so we could get by removing it. What do you think?

Reply:
I think it's because of the new way we're exporting (`export *`)

Reply:
We do use it 3 times in Button.tsx, but it's just wrapping it in a span with a class so removing and just using the markup directly makes sense.

Reply:
>  was talking to Dan about this and he mentioned IconWrapper was added for testing purposes, so we could get by removing it.

I rejigged the test so we no longer needed to export IconWrapper recently https://github.com/Shopify/polaris-react/commit/bdb8432b1e70695603a0fd5daa2fd02957182c26#diff-6735439d743b77b7470242ae481be318


It's local only to this file now so up you to you folks if we want to replace its usage inline or move it into the components directory

Reply:
Just removed it 👍